core/build.gradle

```diff
@@ -7,6 +7,7 @@ test {
 }
 
 dependencies {
+    compile 'javax.inject:javax.inject:1' // @Named is used by the default contract.
```

Comment:
In another change, I'll add `@Param` to 7.x and master. That way, we'll no longer have a hard dep on `@javax.inject.Named`. Maybe we'll remove support for `@javax.inject.Named` in 9.x.